src/option.ts

```diff
@@ -1,6 +1,7 @@
 export interface Options {
   unary_rpc_promise: boolean;
   grpc_package: string;
+  no_namespace: boolean
```

Comment:
Lets rename this to `no_toplevel_namespace`

Reply:
Going to resolve this since the scope of this diff changed to all namespaces.